### DISCUSSION TITLE
Release 3.0.0 - Bluetooth/BLE, SDK 1.5.0

### DISCUSSION THREAD

**Discussion by MYCAMEL222** [earlephilhower/arduino-pico#1241] [General]
Thanks for this release....

Can't wait to start testing ideas.

$6 wifi, ble, dual core, 133 mhz....Pretty cool now.

**Comment by FeuerSturm** [comment]
@earlephilhower 

Many thanks for this awesome release, that finally fixed the problems I was having with my project for any
version later than 2.4.1 (constant crashing of the Pico W that I didn't find a reason for as v2.4.1 worked perfect).

Thanks for all the work you're doing, much appreciated!

**Comment by MYCAMEL222** [comment]
I've been able to try all of the examples and everything works great.

I'd like to be able to rename the broadcast id from "PicoW BLE Joystick" to "My xyz device"....

Is there an easy way to do that?

**Reply by earlephilhower** [reply]
Wait for 3.1, I'm slowly refactoring the HID devices to let stuff like that happen.  OTW, you can hack the libraries, it's pretty obvious what/where to change.

**Reply by MYCAMEL222** [reply]
Thanks for the reply.  I was thinking about doing the hack thing, but didn't want to reinvent something if there was an easy way.

I read as much of the BTstack docs I could take.  I know my limits...I'll wait for 3.1

Thanks for your work!!!!!!!!!!!
